dna/course/zomes/courses/code/src/lib.rs

```diff
@@ -102,11 +102,84 @@ mod courses {
         course::handlers::get_my_enrolled_courses()
     }
 
-    // Section
+    //  ====================== Section definitions
     // TODO: implement section entry definitions
-    // TODO: implement section CRUD methods
+    #[entry_def]
+    fn section_entry_definition() -> ValidatingEntryType {
+        course::entry::section_entry_def()
+    }
+
+    #[entry_def]
+    fn section_anchor_definition() -> ValidatingEntryType {
+        course::anchor::section_anchor_def()
+    }
+
+    // TODO: implement section CRUD methods --> fished out of modules "impl" right? or handlers?
+    #[zome_fn("hc_public")]
+    fn create_section(title: String, course_anchor_address: Address, timestamp: u64, anchor_address: Address,
+    ) -> ZomeApiResult<Address> {
+        section::handlers::create(title, course_anchor_address, timestamp, anchor_address)
+    }
 
-    // Content
+    #[zome_fn("hc_public")]
+    fn get_latest_section_entry(
+        section_anchor_address: Address,
+    ) -> ZomeApiResult<Option<section::entry::Section>> {
+        section::handlers::get_latest_section_entry(section_anchor_address)?;
+    }
+    // this is copied off the course methods code and changed... is the sections_address part relevant here or in the above code?
+    #[zome_fn("hc_public")]
+    fn update_section(
+        title: String,
+        course_anchor_address: Address,
+        timestamp: u64,
+        anchor_address: Address,
+    ) -> ZomeApiResult<Address> {
+        section::handlers::update(title, course_anchor_address, timestamp, anchor_address)
+    }
```

Comment:
hi josh! For this one, we will need the section_anchor_address only and not the course_anchor_address! We don't need to change anything in the course since course entry stores the anchor address of the section. This means that no matter how many times we update the section entry, the section_anchor_address stored in the course entry will always point to the latest version of the section entry! 
With this my suggestion would be,
```suggestion
    fn update_section(
        title: String,
        section_anchor_address: Address,
        timestamp: u64,
    ) -> ZomeApiResult<Address> {
        section::handlers::update(title, section_anchor_address, timestamp)
    }
```